app/editor/package.json

```diff
@@ -25,6 +25,7 @@
     "dequal": "2.0.3",
     "formik": "2.2.9",
     "history": "5.3.0",
+    "html-react-parser": "^3.0.16",
```

Comment:
We don't use a range in the app because it results in different versions for different builds.